content/blog/2018/01/2018-01-26-fosdem-hackaton.adoc

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "You're invited to the Post-FOSDEM 2018 Jenkins Hackfest"
+tags:
+- fosdem
+- event
+author: lnewman
+---
+
+On the first weekend in February numerous free and open source developers from around the
+world will travel to Brussels, Belgium, for arguably the largest event of its kind:
+link:https://fosdem.org[FOSDEM].
+Among the thousands of hackers in attendance will be a number of Jenkins contributors.
+
+On the Monday after FOSDEM,
```

Comment:
What about, "you are invited to join a group of those contributors for a full day of hacking on Jenkins"?

It's a little tighter but I'm not sure...

Reply:
@StackScribe - So:
"On the Monday after FOSDEM, you are invited to join a group of those contributors for a full day of hacking on Jenkins."

Reply:
(bee)